gradio/interface.py

```diff
@@ -139,8 +139,8 @@ def __init__(
         theme (str): Theme to use - one of "default", "huggingface", "grass", "peach". Add "dark" prefix, e.g. "darkpeach" or "darkdefault" for darktheme.
         css (str): custom css or path to custom css file to use with interface.
         allow_screenshot (bool): if False, users will not see a button to take a screenshot of the interface.
-        allow_flagging (bool): if False, users will not see a button to flag an input and output.
-        flagging_options (List[str]): if not None, provides options a user must select when flagging.
+        allow_flagging (str): one of "never", "auto", or "manual". If "never" or "auto", users will not see a button to flag an input and output. If "manual", users will see a button to flag. If "auto", every prediction will be automatically flagged. If "manual", samples are flagged when the user clicks flag button. Can be set with environmental variable GRADIO_ALLOW_FLAGGING.
```

Comment:
Don't think the env variable comment is necessary. I don't think any of our end users would ever need it.

Reply:
I think some of our Gradio Private users use it this way. Anyways, keeping for backwards compatability